include/reactor-uc/scheduler.h

```diff
@@ -1,18 +1,25 @@
 #ifndef REACTOR_UC_SCHEDULER_H
 #define REACTOR_UC_SCHEDULER_H
 
-#include "reactor-uc/queues.h"
 
 typedef struct Scheduler Scheduler;
 typedef struct Environment Environment;
+typedef struct EventQueue EventQueue;
+typedef struct ReactionQueue ReactionQueue;
+
+#include "reactor-uc/queues.h"
 
 struct Scheduler {
-  Environment *env;
-  EventQueue event_queue;
-  ReactionQueue reaction_queue;
+  // methods
   void (*run)(Scheduler *self);
-  void (*prepare_timestep)(Scheduler *self);
+  void (*prepare_time_step)(Scheduler *self);
+
+  // member variables
+  Environment *env_;
```

Comment:
The underscore postfix is to signal that these are private fields which should not be modified directly? I guess this is a fair convention. I think though most (all?) of the fields of these structs would qualify as private since they are only changed in the member methods, if this is the case then I propose to not add this postfix